### ISSUE TITLE
INSTITUTE: Color contrast does not meet minimum requirement in MIIS library search page

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#718]
Color contrast does not meet minimum requirement in highlighted tabs on MIIS library search page https://www.middlebury.edu/institute/academics/library.
